src/expr/type.h

```diff
@@ -531,6 +537,17 @@ class CVC4_PUBLIC SetType : public Type {
   Type getElementType() const;
 }; /* class SetType */
 
+/** Class encapsulating a bag type. */
+class CVC4_PUBLIC BagType : public Type
```

Comment:
It would be best if you did not introduce this, can you try deleting this and everything related to it?

Reply:
Done. Here are the stuff related to [`BagType`](https://github.com/mudathirmahgoub/CVC4/commit/9a91af4e8a0755817fd6689029388b1d376a9882#diff-2b436ded24a9929c65b189d6a5aca8b2L2094).